cmd/config.json

```diff
@@ -21,5 +21,9 @@
 
   "GRPCConnectionTimeoutSeconds": 15,
   "GRPCFunctionTimeoutSeconds": 900,
-  "DAGMode": false
+  "DAGMode": false,
+  "DAGTracePath": "data/traces/sampled_150/20",
```

Comment:
Does this file exist by default?

Reply:
No, a download is required, have added it in the instructions in ```docs/loader.md```